### PR TITLE
fix: (ship ahead 5) apple login issue

### PR DESCRIPTION
**Pull request by uchilaka**
## Jira Card

[STORYAI-###](https://larcity.atlassian.net/browse/STORYAI-###)

## Summary

> Focus on what's important to note here. The Jira card should provide the rest the context.

## Review app(s)

- <storyteller-ui-git-###-donnadieu.vercel.app>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the README/confluence documentation.
- [ ] I have updated the relevant documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/shared/LoadingAnimation.tsx
Comment (uchilaka) [inline src/components/shared/LoadingAnimation.tsx:1]: This can ship ahead in #30

File: src/features/FeatureFlags/index.tsx
Comment (uchilaka) [inline src/features/FeatureFlags/index.tsx:1]: The refactor of  `features/FeatureFlags` can move to a separate PR in the stack

File: src/features/ScriptTag/index.tsx
Comment (uchilaka) [inline src/features/ScriptTag/index.tsx:1]: Implementation of `features/ScriptTag` can ship ahead in #30

**Review by Donnadieu — APPROVED** [review]
